doc/quick-start.md

```diff
@@ -11,13 +10,9 @@ kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
-myst:
-  html_meta:
-    description lang=en: "Quickstart for JupySQL: a package to run SQL in Jupyter"
-    keywords: jupyter, sql, jupysql
-    property=og:locale: en_US
```

Comment:
there are still problems here, you need to copy back the html_meta section into your file: https://raw.githubusercontent.com/ploomber/jupysql/master/doc/quick-start.md and push again